awx/main/tasks.py

```diff
@@ -1179,11 +1179,17 @@ def final_run_hook(self, instance, status, private_data_dir, fact_modification_t
         instance.log_lifecycle("finalize_run")
         job_profiling_dir = os.path.join(private_data_dir, 'artifacts/playbook_profiling')
         awx_profiling_dir = '/var/log/tower/playbook_profiling/'
+        collections_info = os.path.join(private_data_dir, 'artifacts/')
+
         if not os.path.exists(awx_profiling_dir):
             os.mkdir(awx_profiling_dir)
         if os.path.isdir(job_profiling_dir):
             shutil.copytree(job_profiling_dir, os.path.join(awx_profiling_dir, str(instance.pk)))
-
+        if os.path.exists(collections_info):
+            with open(collections_info + 'collections.json') as ee_json_info:
```

Comment:
With my suggestion above this would become: 
```suggestion
            with open(collections_info) as ee_json_info:
```